www/include/configuration/configNagios/nagios.php

```diff
@@ -37,25 +38,28 @@
     exit();
 }
 
-isset($_GET["nagios_id"]) ? $cG = $_GET["nagios_id"] : $cG = null;
-isset($_POST["nagios_id"]) ? $cP = $_POST["nagios_id"] : $cP = null;
-$cG ? $nagios_id = $cG : $nagios_id = $cP;
+$nagiosId = filter_var(
+    $_GET['nagios_id'] ?? $_POST['nagios_id'],
+    FILTER_VALIDATE_INT
+) ?: null;
 
-isset($_GET["select"]) ? $cG = $_GET["select"] : $cG = null;
-isset($_POST["select"]) ? $cP = $_POST["select"] : $cP = null;
-$cG ? $select = $cG : $select = $cP;
+$select = filter_var_array(
+    $_GET["select"] ?? $_POST["select"] ?? [],
+    FILTER_VALIDATE_INT
+);
 
-isset($_GET["dupNbr"]) ? $cG = $_GET["dupNbr"] : $cG = null;
-isset($_POST["dupNbr"]) ? $cP = $_POST["dupNbr"] : $cP = null;
-$cG ? $dupNbr = $cG : $dupNbr = $cP;
+$dupNbr = filter_var_array(
+    $_GET["dupNbr"] ?? $_POST["dupNbr"] ?? [],
+    FILTER_VALIDATE_INT
+);
 
 /* Path to the configuration dir */
 $path = "./include/configuration/configNagios/";
-    
```

Comment:
`$path = "./include/configuration/configNagios/";` could be remove if all usage of $path are removed from formNagios / listNagios